packages/tauri/package.json

```diff
@@ -16,7 +17,8 @@
     "license": "GPL-3.0",
     "devDependencies": {
         "@iarna/toml": "2.2.5",
-        "@tauri-apps/cli": "1.0.0-beta.10",
+        "@tauri-apps/cli": "1.0.0-rc.7",
+        "@tauri-apps/tauricon": "github:tauri-apps/tauricon",
```

Comment:
`tauri icon` appears to be a separate packages now.

Reply:
I would recommend we pin this to a specific tag/version, but since they have none yet, I think this is fine.